integration/container/nat_test.go

```diff
@@ -40,16 +53,26 @@ func TestNetworkNat(t *testing.T) {
 
 func TestNetworkLocalhostTCPNat(t *testing.T) {
 	skip.If(t, testEnv.IsRemoteDaemon)
-	skip.If(t, testEnv.GitHubActions, "FIXME: https://github.com/moby/moby/issues/41561")
```

Comment:
Removed this skip as well; I _think_ this patch should also fix that issue;

- https://github.com/moby/moby/issues/41561